src/zero/module/aou.clj

```diff
@@ -1,23 +1,400 @@
 (ns zero.module.aou
-  "Process Arrays for the All Of Us project.")
+  "Process Arrays for the All Of Us project."
+  (:require [clojure.java.io :as io]
+            [clojure.data.json :as json]
```

Comment:
According to IntelliJ there are a few unused imports here that can be removed (json, set, all & gcs)

Reply:
Nice catch, will do!